debian/README.debian.md

```diff
@@ -0,0 +1,27 @@
+# Build image
+
+To build the image, run:
+```
+make debian
+```
+
+# Run the container
+
+Run the machine with:
+```
+docker run -d linuxbrew/debian tail -f /dev/null
+```
+
+# Enter the container
+
+Use docker-enter to enter the container:
+```
+docker-enter() { boot2docker ssh -t "[ -f /var/lib/boot2docker/nsenter ] || docker run --rm -v /var/lib/boot2docker/:/target jpetazzo/nsenter; sudo /var/lib/boot2docker/docker-enter $@"; }
```

Comment:
Yes, that's macos specific and probably coming from too old documentation. I will remove it.